client/lib/plans/constants.js

```diff
@@ -421,6 +424,16 @@ export const featuresList = {
 		plans: [ PLAN_BUSINESS ]
 	},
 
+	[ FEATURE_LIVE_COURSES ]: {
+		getSlug: () => FEATURE_LIVE_COURSES,
+		getTitle: () => i18n.translate( 'Attend live courses' ),
+		getDescription: () => i18n.translate(
+			"Attend live courses led by Happiness Engineers to get the most out of your site."
+		),
+		getStoreSlug: () => 'live_courses',
```

Comment:
I was curious what `getStoreSlug` is used for, but couldn't find it being called anywhere in Calypso. A full text search returns only this file. Maybe we can leave it out until it's needed?